user_modules/script_runner.lua

```diff
@@ -61,24 +93,55 @@ local function extendedAddTestStep(testStepName, testStepImplFunction, paramsTab
   addTestStep(testStepName, newTestStepImplFunction)
 end
 
---[[ Title + Step approach]]
-local function buildTitle(titleText)
-  local maxLength = 101
-  local filler = "-"
-  local resultTable = {}
-  for line in titleText:gmatch("[^\n]+") do
-    local lineLength = #line
-    if lineLength >= maxLength then
-      table.insert(resultTable, line)
-    else
-      local tailLength = math.fmod(maxLength - lineLength, 2)
-      local emtyLineSideLength = math.floor((maxLength - lineLength) / 2)
-      table.insert(resultTable, filler:rep(emtyLineSideLength) .. line .. filler:rep(emtyLineSideLength + tailLength))
+local function isTestApplicable(testApplicableSdlSettings)
+  if next(testApplicableSdlSettings) == nil then
+   return true
+  end
+  local isMatched = true
+  for _, sdlSettingsSet in pairs(testApplicableSdlSettings) do
+    for sdlSettingsItem, listOfValuesForItem in pairs(sdlSettingsSet) do
+      local sdlBuildOptionValue = Test.sdlBuildOptions[sdlSettingsItem]
+      if sdlBuildOptionValue then
+        if not existsInList(listOfValuesForItem, sdlBuildOptionValue) then
+          isMatched = false
+          break
+        end
+      else
+        return error("SDL build option " .. sdlSettingsItem .. " does not specified!")
+      end
+    end
+    if isMatched then
+      return true
     end
+    isMatched = true
   end
-  return table.concat(resultTable, "\n")
+  return false
+end
+
+local function skipTest(reason)
+  title = ""
+  runner.Title("TEST SKIPPED")
+  runner.Step(
+      "Skip reason",
+      function(skipReason, self)
+        commonFunctions:userPrint(consts.color.cyan, skipReason)
+        self:SkipTest()
+      end,
+      { reason }
+    )
+end
+
+local function printTestInformation()
+  local maxLength = 101
+  local filler = "="
+  print(filler:rep(maxLength) ..
+      "\nDescription:\n" ..
+      runner.testSettings.description ..
+      "\nTest severity: " .. runner.testSettings.severity .. "\n" ..
```

Comment:
@aderiabin Propose to change "Test severity" -> "Severity"

Reply:
@dboltovskyi Rephrased in e1265d7